src/attributes.jl

```diff
@@ -960,8 +960,7 @@ A model attribute for the objective sense of the objective function, which
 must be an `OptimizationSense`: `MIN_SENSE`, `MAX_SENSE`, or
 `FEASIBILITY_SENSE`. The default is `FEASIBILITY_SENSE`.
 
-When setting the sense to `FEASIBILITY_SENSE`, the [`ObjectiveFunction`](@ref)
-attribute must be erased.
+When setting the sense to `FEASIBILITY_SENSE`, the optimizer must erase the [`ObjectiveFunction`](@ref) attribute.
```

Comment:
I would add what it means, e.g., if you set `FEASIBILITY_SENSE` and the `MIN_SENSE` then the objective function should be zero.